app/components/primer/time_ago_component.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Primer
+  # Use Primer::TimeAgoComponent to display a time relative to how long ago it was. This component requires JavaScript.
```

Comment:
Might we want to link to some sort of documentation on how to load the JS?

Reply:
Let's tackle this in a follow up PR for all elements requiring JS